sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateOctKeyOptions.java

```diff
@@ -5,21 +5,51 @@
 import java.time.OffsetDateTime;
 import java.util.Map;
 
+/**
+ * Represents the configurable options to create an AES key.
+ */
 public class CreateOctKeyOptions extends CreateKeyOptions {
+    /**
+     * The AES key size.
+     */
+    private Integer keySize;
```

Comment:
For RSA this called the parent, which also has these options already. If not exposed by the parent, great, but if so they should be inheritted.